src/field/pokemon.ts

```diff
@@ -425,6 +427,24 @@ export abstract class Pokemon extends Phaser.GameObjects.Container {
     this._teraType = value;
   }
 
+  /**
+   * Toxic damage is `1/16 max HP * toxicTurnCount`; increases by 1 per turn.
+   * Ignored if the effect is not {@linkcode StatusEffect.TOXIC}
+   * @defaultValue 0
+   */
+  public get toxicTurnCount(): number {
+    return this.status?.toxicTurnCount ?? 0;
+  }
+
+  /**
+   * The pokemon wakes up when this is `0` and the {@linkcode effect} is {@linkcode StatusEffect.SLEEP}.
+   * Ignored if the effect is not sleep.
```

Comment:
NIT: Match `@linkcode` from `toxicTurnCount`:
```suggestion
   * Ignored if the effect is not {@linkcode StatusEffect.SLEEP}.
```

Reply:
That's already in the previous line which is why I didn't add it again there.